src/test/java/pl/byczazagroda/trackexpensesappbackend/service/WalletGetServiceImplTest.java

```diff
@@ -117,22 +117,29 @@ void shouldNotReturnWallet_WhenWalletByIdNotFound() {
     //fixme, new issue, required improve method for wallets
     @Test
     @DisplayName("when finding wallet by name should return all wallets contains this name pattern")
-    @Disabled
     void shouldReturnAllWalletsContainsNamePattern_WhenFindingWalletByName() {
         // given
         String walletNameSearched = "Family";
         List<Wallet> walletList = createListOfWalletsByName("Family wallet", "Common Wallet", "Smith Family Wallet");
         List<WalletDTO> walletListDTO = walletList.stream().map((Wallet x) ->
-                new WalletDTO(x.getId(), x.getName(), x.getCreationDate(), x.getUser().getId())).toList();
+                new WalletDTO(x.getId(), x.getName(), x.getCreationDate(), null)).toList();
         given(walletRepository.findAll()).willReturn(walletList);
+        given(walletRepository.findAllByNameIsContainingIgnoreCase(walletNameSearched))
+                .willReturn(walletList.stream()
+                                .filter(wallet -> wallet.getName().contains(walletNameSearched))
+                                .toList());
         walletList.forEach(wallet -> given(walletModelMapper.mapWalletEntityToWalletDTO(wallet)).willReturn(
-                walletListDTO.stream().filter(walletDTO -> Objects.equals(wallet.getName(), walletDTO.name())).findAny().orElse(null)));
-
+                walletListDTO
+                        .stream()
+                        .filter(walletDTO -> Objects.equals(wallet.getName(), walletDTO.name()))
```

Comment:
```suggestion
                        .filter(wallet -> wallet.getName().toUpperCase().contains(walletNameSearched.toUpperCase()))
```
In this method, we should compare the names case-insensitive.

Reply:
Please fix tests in `FindAllWalletsByNameCaseSensitiveIT` class.
In the method `createListTestWallets` add another wallet 
```java
        final Wallet fifthWallet = Wallet.builder()
                .user(user)
                .creationDate(Instant.now())
                .name("Wallet Test")
                .build();
```